testsuite/integration-arquillian/tests/base/src/test/resources/export/partialexport-testrealm.json

```diff
@@ -904,7 +904,7 @@
                   "groupOfNames"
                 ],
                 "client.id": [
-                  "finance"
+                  "test-app"
```

Comment:
`PartialExportTest` was failing because of this mapper trying to reference a client that does not exist in the test realm. It seems the mapper entirely can be removed without impact the test.